src/Components/CaseSummaryCard/CaseSummaryCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { router } from 'expo-router';
+import React from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/Entypo';
+
+import styles from './styles';
+import { Case } from '../../types/types';
+
+export default function CaseSummaryCard(caseData: Case) {
+  return (
+    <TouchableOpacity
+      onPress={() =>
+        router.push({
+          pathname: `/Cases/CaseSummaryScreen`,
+          params: {
+            blurb: caseData.blurb,
+            summary: caseData.summary,
+            image: caseData.image,
+            caseSite: caseData.caseSite,
+            date: caseData.date,
+            lawFirm: caseData.lawFirm,
+          },
+        })
+      }
+    >
+      <View style={styles.summaryContainer}>
+        <Image
+          style={styles.imageContainer}
+          source={{
+            uri: caseData.image,
+          }}
+        />
+        <View style={styles.titleContainer}>
+          <Text style={styles.blurbText}>{caseData.blurb}</Text>
+        </View>
+        <View style={styles.bottomContainer}>
+          <View style={styles.inLineInfo}>
+            <Text style={styles.subText}>{String(caseData.date)}</Text>
```

Comment:
Bit of a nit but let's try to get text spacing like it is on Figma. It might be easier to place the first two together as one string of text with the "." in between!